### PR TITLE
[luci] Introduce node accessors to QuantizedModelVerifier test graphs

### PR DESCRIPTION
**Pull request by dayo09**
This commit introduces node accessors to TestGraph classes.

ONE-DCO-1.0-Signed-off-by: Dayoung Lee <dayg502@gmail.com>

----

From #6558

### CODE REVIEW

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 1]
```diff
@@ -452,6 +452,9 @@ template <class Op> class BinaryLogicalOpTestGraph final : public luci::test::Te
     set_minmax_to_non_const(g(), -1, 1);
   }
 
+  loco::Node *x() { return _op->x(); }
+  loco::Node *y() { return _op->y(); }
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:456]: if usage is readonly, 
```cxx
  const loco::Node *x(void) const { return _op->x(); }
  const loco::Node *y(void) const { return _op->y(); }
```

**Comment by seanshpark** [comment]
build error --> I think we may have to go with
```cxx
  loco::Node *x(void) const { return _op->x(); }
  loco::Node *y(void) const { return _op->y(); }
```

**Review by jinevening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**